tests/stream/test_stream_smoke/0034_emit_mode.yaml

```diff
@@ -0,0 +1,318 @@
+          - ['2024-01-24 10:00:00+00:00', '2024-01-24 10:00:10.001000+00:00', 1, 3, 3]
+
+  - id: 3
+    tags:
+      - emit periodic on update
+      - global aggr
+      - window aggr
+    name: aggr emit periodic on update
+    description: Emit global/window aggr results periodic on update
+    steps:
+      - statements:
+          - client: python
+            query_type: table
+            query: drop stream if exists test_35_stream;
+
+          - client: python
+            query_type: table
+            exists: test_35_stream
+            exists_wait: 2
+            wait: 1
+            query: create stream test_35_stream(i int);
+
+          - client: python
+            query_id: emit-mode-1-1
+            query_type: stream
+            depends_on_stream: test_35_stream
+            wait: 1
+            query: select min(i), max(i), count() from test_35_stream emit periodic 1s on update;
+
+          - client: python
+            query_id: emit-mode-1-2
+            query_type: stream
+            depends_on_stream: test_35_stream
+            wait: 1
+            query: select i, count() from test_35_stream group by i emit periodic 1s on update;
+
+          - client: python
+            query_id: emit-mode-1-3
+            query_type: stream
+            depends_on_stream: test_35_stream
+            wait: 1
+            query: select window_start, window_end, min(i), max(i), count() from tumble(test_35_stream, 10s) group by window_start, window_end emit periodic 1s on update;
+
+          - client: python
+            query_id: emit-mode-1-4
+            query_type: stream
+            depends_on_stream: test_35_stream
+            wait: 1
+            query: select window_start, window_end, min(i), max(i), count() from hop(test_35_stream, 10s, 30s) group by window_start, window_end emit periodic 1s on update;
+
+          - client: python
+            query_id: emit-mode-1-5
+            query_type: stream
+            depends_on_stream: test_35_stream
+            wait: 1
+            query: select window_start, window_end, min(i), max(i), count() from session(test_35_stream, 5m) group by window_start, window_end emit periodic 1s on update;
+
+          - client: python
+            depends_on: emit-mode-1-5
+            query_type: table
+            wait: 2
+            query: insert into test_35_stream(i, _tp_time) values(1, '2024-01-24 10:00:00');
+
+          - client: python
+            query_type: table
+            wait: 1
+            query: insert into test_35_stream(i, _tp_time) values(2, '2024-01-24 10:00:05');
+
+          - client: python
+            query_type: table
+            kill: emit-mode-1-1,emit-mode-1-2,emit-mode-1-3,emit-mode-1-4,emit-mode-1-5
+            kill_wait: 2
+            wait: 1
+            query: insert into test_35_stream(i, _tp_time) values(3, '2024-01-24 10:00:10');
+
+    expected_results:
+      - query_id: emit-mode-1-1
+        expected_results:
+          - [1, 1, 1]
```

Comment:
What's the difference between `emit periodic and emit periodic on update` ? From the testing results, they look the same ?

Reply:
will new commit to tweak the behavior